runner.go

```diff
@@ -362,6 +363,13 @@ func applyFormatTemplate(contents, key string) (string, error) {
 	return buf.String(), nil
 }
 
+func replaceKey(args ...interface{}) string {
```

Comment:
The passed in type probably should be `...string` to make it a list of strings instead of '...interface'. Template functions support more specific types and you assert them all to strings anyways.